server/matchmaker/search.py

```diff
@@ -64,38 +65,34 @@ def ratings(self):
         ratings = []
         for player, rating in zip(self.players, self.raw_ratings):
             # New players (less than config.NEWBIE_MIN_GAMES games) match against less skilled opponents
-            if player.ladder_games <= config.NEWBIE_MIN_GAMES and self.rating_prop == 'ladder_rating':
+            if (player.ladder_games <= config.NEWBIE_MIN_GAMES
+                    and self.rating_type is RatingType.LADDER_1V1):
                 rating = self.adjusted_rating(player)
             ratings.append(rating)
         return ratings
 
     @property
     def raw_ratings(self):
-        return [getattr(player, self.rating_prop) for player in self.players]
+        return [player.ratings[self.rating_type] for player in self.players]
 
-    @property
-    def boundary_80(self):
+    def _nearby_rating_range(self, delta):
         """
-        Returns 'boundary' mu values for achieving roughly 80% quality
-
-        These are the mean, rounded to nearest 10, +/- 200, assuming sigma <= 100
+        Returns 'boundary' mu values for player matching. Adjust delta for
+        different game qualities.
         """
-        # TODO: Figure out what to do with these boundaries
         mu, _ = self.ratings[0]  # Takes the rating of the first player, only works for 1v1
-        rounded_mu = int(math.ceil(mu / 10) * 10)
-        return rounded_mu - 200, rounded_mu + 200
+        rounded_mu = int(math.ceil(mu / 10) * 10) # Round to 10
+        return rounded_mu - delta, rounded_mu + delta
 
     @property
-    def boundary_75(self):
-        """
-        Returns 'boundary' mu values for achieving roughly 75% quality
+    def boundary_80(self):
+        """ Achieves roughly 80% quality. """
+        return self._nearby_rating_range(200)
 
-        These are the mean, rounded to nearest 10, +/- 100, assuming sigma <= 200
-        """
-        # TODO: Figure out what to do with these boundaries
-        mu, _ = self.ratings[0]  # Takes the rating of the first player, only works for 1v1
-        rounded_mu = int(math.ceil(mu / 10) * 10)
-        return rounded_mu - 100, rounded_mu + 100
+    @property
+    def boundary_75(self):
+        """ Achieves roughly 75% quality. FIXME - why is it MORE restrictive??? """
```

Comment:
I guess my TODO comment was poorly worded. But how are these boundaries going to make sense in a team context?

Reply:
Dunno, I didn't want to change behaviour here yet.